docs/features/software-catalog/catalog-customization.md

```diff
@@ -5,64 +5,159 @@ title: Catalog Customization
+
+```tsx title="packages/app/src/App.tsx"
+<Route
+  path="/catalog"
+  element={<CatalogIndexPage initiallySelectedFilter="all" />}
+/>
+```
+
+Possible options are: owned, starred, or all
+
+## Initially Selected Kind
+
+By default the initially selected Kind when viewing the Catalog in Component, but you may have reasons that you want this to be different. Let's say at your Organization they would like it to always default to Domain, here's how you would do that:
+
+```tsx title="packages/app/src/App.tsx"
+<Route path="/catalog" element={<CatalogIndexPage initialKind="domain" />} />
+```
+
+Possible options are all the [default Kinds](system-model.md) as well as any custom Kinds that you have added.
+
+## Owner Picker Mode
+
+The Owner filter by default will only contain a list of Users and/or Groups that actually own an entity in the Catalog, now you may have reason to change this. Here's how:
+
+```tsx title="packages/app/src/App.tsx"
+<Route path="/catalog" element={<CatalogIndexPage ownerPickerMode="all" />} />
+```
+
+Possible options are: owners-only or all
+
+## Customize Columns
+
+By default the columns you see in the `CatalogIndexPage` were selected to be a good starting point for most but there may be reasons that you would like to customize these with more or less columns. On primary use case for this customization is if you added a custom Kind. Support for this was added in v1.23.0 of Backstage, make sure you are on that version or newer to use this feature. Here's an example of how to make this customization:
+
+```tsx title="packages/app/src/App.tsx"
+import {
+  CatalogEntityPage,
+  CatalogIndexPage,
+  catalogPlugin,
+  {/* highlight-add-start */}
+  CatalogTable,
+  CatalogTableColumnsFunc,
+  {/* highlight-add-end */}
+} from '@backstage/plugin-catalog';
+
+{/* highlight-add-start */}
+const myColumnsFunc: CatalogTableColumnsFunc = entityListContext => {
+  if (entityListContext.filters.kind?.value === 'MyKind') {
+    return [
+      CatalogTable.columns.createNameColumn(),
+      CatalogTable.columns.createOwnerColumn(),
+    ];
+  }
+
+  return CatalogTable.defaultColumnsFunc(entityListContext);
 };
+{/* highlight-add-end */}
+
+{/* highlight-remove-next-line */}
+<Route path="/catalog" element={<CatalogIndexPage />} />
+{/* highlight-add-next-line */}
+<Route path="/catalog" element={<CatalogIndexPage columns={myColumnsFunc} />} />
 ```
 
-The `EntityListProvider` shown here provides a list of entities from the
-`catalog-backend`, and a way to hook in filters.
+> Note: the above example has been simplified and you will most likely have more code then just this in your `App.tsx` file.
 
-Now we're ready to create a new filter that implements the `EntityFilter`
-interface:
+## Customize Actions
+
+The `CatalogIndexPage` comes with three default actions - view, edit, and star. You might want to add more here's how:
+
+First you'll need to add `@mui/utils` to your `packages/app/package.json`:
```

Comment:
```suggestion
The `CatalogIndexPage` comes with three default actions - view, edit, and star. You might want to add more.

To do this, first you'll need to add `@mui/utils` to your `packages/app/package.json`:
```

Possible suggestion, just removes the duplicate colon entry points.

Reply:
I like it, changed to match 👍